sources/assets/shells/aliases.d/routersploit

```diff
@@ -0,0 +1,2 @@
+alias rsf='rsf.py'
```

Comment:
routersploit is installed through pipx and the authors configured it in a way it's installed as `rsf.py`, the alias is here for people that would try `rsf` only
I guess we could remove that alias, don't know what's best 🤷

Reply:
Ok, that makes sense then if user are use without the `.py` i can understand
(but if thats the way it's installed it could be confusing too)
Maybe we should removed this one and update the test command accordingly (if needed)

Reply:
I'm removing it for now to keep only the `official` bin command but we could add it later if needed. (to avoid multiple command to the same bin)